main/cicd/cicd-pipeline/config/infra/cloudformation.yml

```diff
@@ -199,6 +199,115 @@ Resources:
+            Resource: !Sub 'arn:aws:iam::${AWS::AccountId}:policy/${self:custom.settings.namespace}-*'
+          - Action:
+              - dynamodb:*
+            Effect: Allow
+            Resource: !Sub 'arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/${self:custom.settings.namespace}-*'
+          - Effect: Allow
+            Action:
+              - logs:CreateLogStream
+              - logs:DescribeLogStreams
+              - logs:PutLogEvents
+              - logs:CreateLogGroup
+              - logs:DeleteLogGroup
+              - logs:DeleteLogStream
+            Resource:
+              - !Sub 'arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/*'
+              - !Sub 'arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/*:log-stream:*'
+          - Effect: Allow
+            Action:
+              - s3:*
+              - lambda:*
+              - apigateway:*
+              - cloudfront:*
+              - servicecatalog:*
+            Resource:
+              - '*'
+          - Effect: Allow
+            Action:
+              - ssm:GetParameter
+            Resource:
+              - !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${self:custom.settings.envName}/*'
+          - Effect: Allow
+            Action:
+              - iam:ListRoles
+              - organizations:DescribeOrganization
+              - account:ListRegions
+              - cloudformation:ValidateTemplate
+            Resource:
+              - '*' # For the actions listed above IAM does not support resource-level permissions and requires all resources to be chosen
+          - Effect: Allow
+            Action:
+              - iam:CreateServiceLinkedRole
+              - iam:DeleteServiceLinkedRole
+            Resource:
+              - !Sub 'arn:aws:iam::${AWS::AccountId}:role/aws-service-role/replicator.lambda.amazonaws.com/AWSServiceRoleForLambdaReplicator'
+              - !Sub 'arn:aws:iam::${AWS::AccountId}:role/aws-service-role/logger.cloudfront.amazonaws.com/AWSServiceRoleForCloudFrontLogger'
+          - Effect: Allow
+            Action:
+              - codebuild:CreateReportGroup
+              - codebuild:CreateReport
+              - codebuild:BatchPutTestCases
+              - codebuild:UpdateReport
+            Resource:
+              - '*'
+          - Effect: Allow
+            Action:
+              - cloudformation:CreateStack
+              - cloudformation:DeleteStack
+              - cloudformation:DescribeStacks
+              - cloudformation:UpdateStack
+              - cloudformation:GetTemplate
+              - cloudformation:ListStackResources
+              - cloudformation:DescribeStackEvents
+            Resource:
+              - !Sub 'arn:aws:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/${self:custom.settings.namespace}-*'
+              - !Sub 'arn:aws:cloudformation:us-east-1:${AWS::AccountId}:stack/${self:custom.settings.envName}-va-${self:custom.settings.solutionName}-*'
```

Comment:
Is there a reason us-east-1 follows a different naming conventions here?

Reply:
No matter which region SWB is deployed in, the stack for EdgeLambda will always be deployed in `us-east-1`. That solves the question as to why that instead of `${AWS::Region}`.

As for why the string coming after `stack/`, namespace is constructed as `<envName>-<regionShortName>-<solutionName>`. Since we know we're looking at `us-east-1` on this line the original namespace might not apply, so we need to break it down.

Reply:
Gotcha! That makes sense.